spec/content/kibana/spec.yml

```diff
@@ -29,3 +29,8 @@ spec:
       pattern: '^{PACKAGE_NAME}-.+\.json$'
       forbiddenPatterns:
         - '^.+-(ecs|ECS)\.json$' # ECS suffix is forbidden
+  - description: Folder containing Kibana knowledge base entries
+    type: folder
+    name: kb_entry
+    required: false
+    $ref: "../../integration/kibana/kb_entry/spec.yml"
```

Comment:
So, I decided to make knowledge base entries (first-class) citizens of the `kibana` spec instead of having them live in their own, dedicated root folder. 

The reasoning is that:
1. KB entries will be "managed" by Kibana 
1. KB entries will only be used via Kibana (at least for now)

So I felt like it made more sense to have it done that way.

Reply:
> Do you foresee other uses of this data?

We **might** want at some point to expose this data more directly to the end user, and make is less of an implementation detail. But we don't have any use case for this right now, so I think we should just ignore it for now and eventually revisit later.